lua/plugins/conform.lua

```diff
@@ -42,7 +42,7 @@ return {
 		},
 
 		-- Set up format-on-save
-		format_on_save = { timeout_ms = 3000 },
+		--format_on_save = { timeout_ms = 3000 },
```

Comment:
```suggestion
		format_on_save = { timeout_ms = 3000 },
```
I would like to keep this feature